include/crm/cluster/compat.h

```diff
@@ -127,18 +129,25 @@ guint crm_active_peers(void);
 //! \deprecated Do not use
 guint reap_crm_member(uint32_t id, const char *name);
 
+// NOTE: sbd (as of at least 1.5.2) uses this enum
 //!@{
 //! \deprecated Use <tt>enum pcmk_cluster_layer</tt> instead
 enum cluster_type_e {
+    // NOTE: sbd (as of at least 1.5.2) uses this value
```

Comment:
sbd also still uses `pcmk_cluster_` values that are only available in `compatibility.h`, if that matters to you; they're inside CPP guards

I'm not checking all of these nor am I checking that the comments are exhaustive. I imagine you were thorough. (There are so many of these, I am curious how you did it, especially given that some of them like "online" are so generic-looking and likely to generate false positives.)

Reply:
> sbd also still uses `pcmk_cluster_` values that are only available in `compatibility.h`, if that matters to you; they're inside CPP guards

That's for building with ancient versions of Pacemaker. It doesn't include compatibility.h, so it's not using it with current Pacemaker code.


> I'm not checking all of these nor am I checking that the comments are exhaustive. I imagine you were thorough. (There are so many of these, I am curious how you did it, especially given that some of them like "online" are so generic-looking and likely to generate false positives.)

I deleted the Pacemaker includes from sbd source files and saw what broke :)

More precisely: delete an include, see the first missing symbol reported, define that symbol trivially in the file, see what the next missing symbol reported is, etc. Repeat until no more missing symbols